science/gmt6/Portfile

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.0
+PortGroup           github 1.0
+
+name                gmt6
+github.setup        GenericMappingTools gmt 6.5.0
+github.tarball_from releases
+distname            ${github.project}-${github.version}-src
+revision            2
+epoch               1
+
+categories          science
+maintainers         {me.com:remko.scharroo @remkos} openmaintainer
+license             GPL-3
+
+description         The Generic Mapping Tools
+long_description GMT is an open source collection of ~120 tools \
+    for manipulating geographic and Cartesian data sets and     \
+    producing PostScript illustrations ranging from simple x-y  \
+    plots via contour maps to artificially illuminated surfaces \
+    and 3D perspective views.
+
+homepage            https://www.generic-mapping-tools.org/
+master_sites        https://github.com/GenericMappingTools/gmt/releases/download/${version} \
+                    ftp://ftp.soest.hawaii.edu/gmt          \
+                    ftp://ftp.star.nesdis.noaa.gov/pub/sod/lsa/gmt \
+                    ftp://ftp.iris.washington.edu/pub/gmt   \
+                    ftp://ftp.iag.usp.br/pub/gmt            \
+                    ftp://gmt.mirror.ac.za/gmt              \
+                    http://mirrors.ustc.edu.cn/gmt          \
+                    http://www.scc.u-tokai.ac.jp/gmt
+use_xz              yes
+distname            gmt-${version}
+distfiles           ${distname}-src${extract.suffix}
+
+checksums           rmd160  3e47113d134bde756b98906a4704b3151902c4d1 \
+                    sha256  4022adb44033f9c1d5a4d275b69506449e4d486efe2218313f3ff7a6c6c3141e \
+                    size    58696516
+
+patchfiles          patch-gmt6-common_sighandler.diff
+
+depends_lib         port:curl \
+                    port:dcw-gmt \
+                    port:ghostscript \
+                    port:gshhg-gmt \
+                    port:netcdf
+
+default_variants    +gdal +pcre
+if {![variant_isset lgpl]} {
+    default_variants-append +nonfree
+}
+
+cmake.out_of_source yes
```

Comment:
this can be removed as it is the default in tue latest PortGroup version.

Reply:
Removed in commit  c70ef3c

Reply:
It's still there

Reply:
Oops. You're right. I fixed it in gmt5/Portfile but overlooked it in gmt6/Portfile.
Fixed now in [3b87f0f](https://github.com/macports/macports-ports/commit/3b87f0fd3cc6db67fe5f3fbf67f9ce0a59c21dc3)